projects/addon-doc/components/navigation/navigation.component.ts

```diff
@@ -165,6 +164,16 @@ export class TuiDocNavigation {
         this.openActivePageGroup();
     }
 
+    protected onFocusSearch(event: KeyboardEvent): void {
+        if (
+            event.code === 'Slash' &&
```

Comment:
Would `(window:keydown.slash)` work?

Reply:
No, I put together a [stackblitz ](https://stackblitz.com/edit/angular-6xnn6k?file=src%2Fapp%2Fapp.component.ts)to demonstrate